.storybook/stories/datepicker/datepicker.stories.ts

```diff
@@ -81,7 +76,7 @@ export const Datepicker: StoryObj = {
 export const DefaultDate: StoryObj = {
   render: DatePickerTemplate,
   args: {
-    clrDate: '2022-01-01 00:00:00.000',
+    clrDate: 1641038400000,
```

Comment:
Please undo this change. I changed this be more readable in #1490 and we shod keep this change.